content/en/docs/Examples/boot_assessment_trusted_boot.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Enabling Automatic Boot Assessment with Trusted Boot"
+linkTitle: "Enabling Automatic Boot Assessment with Trusted Boot"
+weight: 4
+description: This section describe examples on how to enable Automatic Boot Assessment with Trusted Boot in your own services.
+---
+
+
+# Enabling Automatic Boot Assessment with Trusted Boot
```

Comment:
I think that by having also the title here it will display it twice, not 100% sure, but we don't do it in other parts too indeed: https://raw.githubusercontent.com/kairos-io/kairos-docs/refs/heads/main/content/en/docs/Examples/k3s-stages.md

```suggestion
```